src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/CompatDifference.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.CodeAnalysis;
+using System;
+
+namespace Microsoft.DotNet.ApiCompatibility.Abstractions
+{
+    public class CompatDifference : IDiagnostic, IEquatable<CompatDifference>
```

Comment:
we should probably add some triple slash comments to the public types

Reply:
Good call. I'll work on that.